### DISCUSSION TITLE
unable to log into moonraker SOLVED

### DISCUSSION THREAD

**Discussion by jasonsaffle13** [Arksine/moonraker#482] [General]
https://pastebin.ubuntu.com/p/HTn4d48Mr2/
trying to get Can working I see the network but can not log into Moonraker.

**Comment by Arksine** [comment]
It looks like either Moonraker's source code was modified or your sdcard is corrupted.  Unfortunately its a core component, so Moonraker cannot start without it.  You'll have to ssh into to correct it:
```
cd ~
sudo service moonraker stop
rm -rf moonraker
git clone https://github.com/Arksine/moonraker.git
sudo service moonraker start
```

**Comment by jasonsaffle13** [comment]
Thank you!! that worked!!
